kernel/demoProjects/build.gradle

```diff
@@ -0,0 +1,20 @@
+version 'unspecified'
```

Comment:
yea this is good and also please make the other demo jars work this way.

Reply:
I added tablesawDemo project